cranelift/codegen/src/isa/x64/inst/emit.rs

```diff
@@ -2820,25 +2819,6 @@ pub(crate) fn emit(
             one_way_jmp(sink, CC::NZ, again_label);
         }
 
-        Inst::Fence { kind } => {
-            sink.put1(0x0F);
-            sink.put1(0xAE);
-            match kind {
-                FenceKind::MFence => sink.put1(0xF0), // mfence = 0F AE F0
-                FenceKind::LFence => sink.put1(0xE8), // lfence = 0F AE E8
-                FenceKind::SFence => sink.put1(0xF8), // sfence = 0F AE F8
-            }
-        }
-
-        Inst::Hlt => {
-            sink.put1(0xcc);
```

Comment:
Just wanted to flag that we will now be using a new encoding here: `0xf4` versus `0xcc` originally. `0xCC` is the encoding for `INT3`.

Reply:
Oh good catch! I'll make sure the encoding doesn't change